### ISSUE TITLE
[RFC]: add `math/base/special/sind`

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/stdlib#32]
## Checklist

> Please ensure the following tasks are completed before filing an issue.

* [x] Read and understood the [Code of Conduct][code-of-conduct].
* [x] Searched for existing issues and pull requests.
* [x] If this is a general question, searched the [FAQ][faq] for an existing answer.
* [x] If this is a feature request, the issue name begins with `RFC: `.

## Description

> Description of the issue (or feature request).

Compute the sine of `x` where `x` is in degrees.

Package: `@stdlib/math/base/special/sind`
Alias: `base.sind`

For a reference implementation, see [Julia](https://github.com/JuliaLang/julia/blob/c3f59178b1c9c2b0a9758ac4f92b3445e933e405/base/special/trig.jl#L478).

## Related Issues

> Does this issue (or feature request) have any related issues?

No.

## Questions

> Any questions for reviewers?

No.

## Other

> Any other information relevant to this issue (or feature request)? This may include screenshots, references, stack traces, sample output, and/or implementation notes.

No.

[code-of-conduct]: https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md
[faq]: https://github.com/stdlib-js/stdlib/blob/develop/FAQ.md

**Comment by rreusser** [comment]
Are the degree modules one-liners that multiply by pi / 180 and call out to other functions?

**Comment by kgryte** [comment]
Not necessarily, as one can perform argument reduction for better accuracy. See [Julia](https://github.com/JuliaLang/julia/blob/226e2dc026bd190d477608929ac2e8755cc88103/base/special/trig.jl#L351), for example.

**Comment by kgryte** [comment]
Based on the reference implementation, implementing this package depends on kernel `sin`.

cc/ @Planeshifter

**Comment by vinothdinakar** [comment]
I see the above comments on May 6. Is anyone working on this? or can I try ?

**Comment by kgryte** [comment]
No one was yet to work on this as we have yet to implement kernel `sin`. @Planeshifter may have some insight into timelines for this.

**Comment by Planeshifter** [comment]
The kernel `sin` function is already there inside of `@stdlib/math/base/special/sin`, but needs to be separated into its own package. I can make this a high-priority item and work on this soon so that @vinothdinakar can tackle `sind`.

**Comment by kgryte** [comment]
@Planeshifter Looks like we will also need kernel cosine, as well.

**Comment by Planeshifter** [comment]
If you want to start working on this, @vinothdinakar, you could require the internal package functions directly for now, as in

``` javascript
var kernelSin = require( '@stdlib/math/base/special/sin/lib/kernel_sin.js' );
```

We can then replace the require statements once the standalone packages are up. I will have the kernel packages done by the end of the week, at latest.

**Comment by kgryte** [comment]
@Planeshifter Awesome!

@vinothdinakar Feel free to submit a PR once you have a working prototype.

**Comment by vinothdinakar** [comment]
Ok Thanks

**Comment by kgryte** [comment]
@vinothdinakar Just to give you an update, @Planeshifter managed to implement the various kernel packages:

[@stdlib/math/base/special/kernel-sin](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/kernel-sin)
[@stdlib/math/base/special/kernel-cos](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/kernel-cos)

So, if you are still interested and/or already working on it, feel free to use the above packages. :)

Updated reference implementation: https://github.com/JuliaLang/julia/blob/9ef2af9db620644253941e080b5b4986ad0d6091/base/special/trig.jl#L1017

**Comment by abhis-1** [comment]
Hi @kgryte, is this issue still open or someone has completed the task
